oak_containers_system_image/build.sh

```diff
@@ -1,11 +1,15 @@
 #!/bin/bash
 
+mkdir -p target
+
+# build the orchestrator binary
+cargo build --package="oak_containers_orchestrator" --release --target="x86_64-unknown-linux-musl" -Z unstable-options --out-dir="./target"
```

Comment:
I assume the unstable-options is so that you could override out-dir?

Reply:
yes!